lib/tdf3/src/crypto/declarations.ts

```diff
@@ -59,13 +67,15 @@ export type CryptoService = {
    * Generate an RSA key pair
    * @param size in bits, defaults to a reasonable size for the default method
    */
-  generateKeyPair: (size?: number) => Promise<CryptoKeyPair>;
+  generateKeyPair: (size?: number) => Promise<PairT>;
```

Comment:
`PairT` means?

Reply:
I'll add more documentation soon, but this is to be a key pair type - an abstraction of CryptoKeyPair